app-emulation/virtio-win/virtio-win-0.1.173.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="VirtIO drivers for Windows virtual machines running on KVM"
+HOMEPAGE="https://fedoraproject.org/wiki/Windows_Virtio_Drivers"
```

Comment:
This homepage seems outdated.

Reply:
Thanks for noticing, I've added the new page.

The url is a long one, hopefully it's not a problem.